config/vsphere-csi-driver/overwrites/values.yaml

```diff
@@ -10,38 +10,6 @@ config:
 
 controllerDeployment:
   replicas: 1
-  csiAttacher:
```

Comment:
These patches make image and tag configurable but I don't see the point in this as they will remain outdated if we don't go looking manually for the versions in the upstream repo ourselves. I think we should rely on the image tags used in each release.

Reply:
We could replace the most common image repo names with the GS one but this isn't 100% reliable.

Reply:
the only issue i can see here is if we're running in a proxied environment where the customer controls the domain allowlist - having everything pulled from a single location is better

Reply:
Still not configurable in the values but the registry/repo get replaced with our own in https://github.com/giantswarm/cloud-provider-vsphere-app/pull/98/commits/a2deb56218fe94d86f9a2459722bf15dd5260c3b